content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,94 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+## Deploying ModelDB
+
+Use the below commands to deploy ModelDB.
+
+```
+ks generate modeldb modeldb
+ks apply default -c modeldb
+```
+
+## Using ModelDB
+ 
+ModelDB organizes model data in a 3-level model hierarchy. At the bottom of the hierarchy, is an ExperimentRun: every execution of a script/program creates an ExperimentRun.
+Next, related ExperimentRuns can be grouped into an Experiment (e.g., "running hyperparameter optimization for the Neural Network"). Finally, all Experiments and ExperimentRuns belong to a Project (e.g., "churn prediction").
+
+After ModelDB is deployed and modeldb-db, modeldb-backend and modeldb-frontend pods are running - 
+
+1)  Loading Metrics from Python
+
+Open up a Jupyter notebook you will use for model training. Make sure that the ModelDB libraries have been installed already or install via 'pip install modeldb'.
```

Comment:
Can you clarify that this can be a notebook or python file.

Can you make installing modeldb package its own bullet item?

Could also file an issue to add modeldb to our Jupyter runtime images?